skops/io/_numpy.py

```diff
@@ -26,16 +27,11 @@ def ndarray_get_state(obj: Any, save_state: SaveState) -> dict[str, Any]:
             res["type"] = "json"
             res["shape"] = get_state(obj.shape, save_state)
         else:
-            # Memoize the object and then check if it's file name (containing
-            # the object id) already exists. If it does, there is no need to
-            # save the object again. Memoizitation is necessary since for
-            # ephemeral objects, the same id might otherwise be reused.
-            obj_id = save_state.memoize(obj)
+            data_buffer = io.BytesIO()
+            np.save(data_buffer, obj)
+            obj_id = save_state.memoize(obj)  # TODO: useless
```

Comment:
I don't understand why this is now useless.

Reply:
It's useless because no memoization takes place, but I'm working on it.